tools/cockpit.spec

```diff
@@ -331,17 +331,17 @@ The Cockpit components for managing virtual machines.
 
 %files machines -f machines.list
 
-%package ovirt
+%package machines-ovirt
```

Comment:
This at least needs some `Provides:/Obsoletes:` to not break upgrades on Fedora.

Reply:
So this Provides/Obsoletes needs to be guarded to only apply to `if 0%{?fedora} >= 25` so that it doesn't apply to rhel/centos, as you say that the cockpit-ovirt package there should be kept. However, in Fedora it's still needed to not break upgrades.

Reply:
In fact, `cockpit-ovirt` package is not installed, the "other" project [1] recently provides `cockpit-ovirt-dashboard` package instead. The `cockpit-ovirt` is sort of "reserved" name for [1]. 

The root issue here is to avoid user's confusion, since `cockpit-ovirt` [1] project provides `cockpit-ovirt-dashboard` and it would be messy if `cockpit-ovirt` package would come from `cockpit` and not from the `cockpit-ovirt`.

@evol262, can you please comment here?

Reply:
@mareklibra : For RHEL that may be, and hence I *don't* want to apply the Provides/Obsoletes there. But on Fedora 26/27, `cockpit-ovirt` does exist and is *the* package from Cockpit. If we rename it, it does need the P/O: in order to upgrade cleanly.

Reply:
ok, added